genesyscloud/journey_views/resource_genesyscloud_journey_views_schema.go

```diff
@@ -210,3 +212,27 @@ func ResourceJourneyViews() *schema.Resource {
 		},
 	}
 }
+
+func JourneyViewExporter() *resourceExporter.ResourceExporter {
+	return &resourceExporter.ResourceExporter{
+		GetResourcesFunc:        provider.GetAllWithPooledClient(getAllJourneyViews),
+		RefAttrs:                map[string]*resourceExporter.RefAttrSettings{},
```

Comment:
You don't need to include attributes unless you're given them a value so you can remove `RefAttrs`, `RemoveIfMissing` and `CustomAttributeResolver`